src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/DefaultHtmlGenerator.cs

```diff
@@ -338,7 +339,36 @@ public virtual IHtmlContent GenerateAntiforgery(ViewContext viewContext)
                 isChecked: false,
                 setId: true,
                 isExplicitValue: true,
-                format: null,
+                format: format,
+                htmlAttributes: htmlAttributeDictionary);
+        }
+
+        /// <inheritdoc />
+        public virtual TagBuilder GenerateTextBox(
```

Comment:
Please move this back where it was. It's hard to see what, if anything, changed in this method.